lib/pynml/nml.py

```diff
@@ -2401,6 +2448,18 @@ def get_has_link(self):
         return copy(self._has_link_links)
 
 
+class Environment(NMLObject):
+    """
+    Describes attributes inherent to the environment.
+
+    Attributes to be attached to the environment the topology isin..
+    """
+
+    def __init__(
```

Comment:
In fact, a method that only calls ``super`` with the exact same arguments than its ancestor is unnecessary